linkup-cli/src/commands/deploy/resources.rs

```diff
@@ -5,6 +5,7 @@ use sha2::{Digest, Sha256};
 
 use super::{api::CloudflareApi, cf_deploy::DeployNotifier, DeployError};
 
+const LINKUP_ACCOUNT_TOKEN_NAME: &str = "linkup-account-token";
```

Comment:
I wonder if a different name might clarify the way this is used better - like `linkup-account-owned-cli-access-token` ?